huaweicloud/resource_huaweicloud_compute_instance.go

```diff
@@ -471,6 +471,14 @@ func ResourceComputeInstanceV2() *schema.Resource {
 				Deprecated:    "use tags instead",
 				Elem:          &schema.Schema{Type: schema.TypeString},
 			},
+			"created_at": {
```

Comment:
please add the two attribute after `staus`

Reply:
done